CHANGELOG.rst

```diff
@@ -1,5 +1,29 @@
 Changelog
 =========
+1.4.0 (2019-10-30)
+-----------------------
+MLflow 1.4.0 includes several major features:
+
+- Model Registry. MLflow 1.4 adds an experimental model registry feature, where you can manage, version, and keep lineage of your production models. (#1943, @mparkhe, @Zangr, @sueann, @dbczumar, @smurching, @gioa, @clemens-db, @pogil, @mateiz)
+- Keras support for TensorFlow 2.0. MLflow Keras model saving, loading, and logging has been updated to be compatible with TensorFlow 2.0.  (#1927, @juntai-zheng)
+- Autologging for TensorFlow 2.0. Autologging for ``tf.estimator`` and ``tf.keras`` models has been updated to be compatible with TensorFlow 2.0. The same functionalities of autologging in TF 1.x will be offered in TF 2.0, namely when fitting tf.keras models and when exporting saved tf.estimator models. (#1910, @juntai-zheng)
+
+More features and improvements:
+
+- Add functions ``get_run``, ``get_experiment``, ``get_experiment_by_name`` to the fluent API (#1923, @fhoering)
+- Use Plotly as artifact image viewer to make it interactive (#1934, @harupy)
+- Enable scrolling to zoom in metric and run comparison plots (#1929, @harupy)
+- Added support of viewfs URIs for HDFS federation for artifacts (#1947, @t-henri)
+
+Bug fixes and documentation updates:
+
+- Make detection of ``MLproject`` files case-insensitive (#1981, @smurching)
+- Fix encoding of metric names in MLflow UI URLs (#1968, @smurching)
+- Get ``models serve`` CLI command to work in Windows (#1949, @rboyes)
```

Comment:
Get ``models serve`` CLI command to work in Windows  -> ``models serve`` CLI command now works in Windows